heka/init.sls

```diff
@@ -12,4 +12,7 @@ include:
 {%- if pillar.heka.aggregator is defined %}
 - heka.aggregator
 {%- endif %}
+{%- if pillar.heka.os_telemetry_collector is defined %}
```

Comment:
please move it before `- heka.aggregator`. This is because of grains file's generation.